pkgs/darwin-installer/default.nix

```diff
@@ -24,6 +24,9 @@ stdenv.mkDerivation {
   shellHook = ''
     set -e
 
+    orig_path="$PATH"
+    export PATH="${pkgs.openssh}/bin"  # In case nix needs it
```

Comment:
This clears path, also in what case is openssh used?

Reply:
> This clears path

That’s the point. (The later commit add `/usr/bin` and `/bin`, but, honestly, I would prefer not to add them and have an explicit list of macOS binaries used instead.)

> in what case is openssh used?

With ssh substituters. I wonder whether Nix should be using `openssh` from nixpkgs, but currently substitution fails when `openssh` is not in `$PATH`, so, I suppose it invokes `ssh` from `$PATH`.

Reply:
I had an old tab open without the fixup commit.

An ssh substituter like remote builders? Only the nix-daemon uses ssh in that case AFAIK.

Reply:
I only added this line because it was failing in my case 🤷‍♂️.
I don’t use nix-daemon, plain old single-user nix with one ssh substituter configure in addition to cache.nixos.org. It is not really a remote builder (my nix cannot offload a build to it), it is just a remote ssh-store from which it can pull some packages.